src/components/UserProfileCard/UserProfileCard.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./UserProfileCard.scss";
+import staffPicture from "../../assets/images/users/staff-picture.png";
+
+const UserProfileCard = () => {
```

Comment:
Currently takes no prompts so can't be changed when called. Would be great to take a staff name, type of employment, and image as prompts as well as any others you deem necessary